R/testGeno.R

```diff
@@ -12,6 +12,10 @@ testGenoSingleVar <- function(nullmod, G, E = NULL, test = c("Score", "Score.SPA
     test <- match.arg(test)
     calc.score <- test %in% c("Score", "Score.SPA") | (recalc.pval.thresh < 1)
 
+    if (is.null(nullmod$cholSigmaInv) && (calc.score || !is.null(E))) {
+        stop("small null model cannot be used with options provided")
+    }
+
```

Comment:
This should be a function (`isSmallNullModel`?), since it may end up being useful other places.

Reply:
If we're changing `smallNullModel` to `nullModelSmall`, should we call this `isNullModelSmall`?

Reply:
`isNullModelSmall` seems reasonable to me, and makes me laugh. However, I think the check is simply `is.null(nullmod$cholSigmaInv)`.